tensorflow/contrib/tensorrt/test/base_test.py

```diff
@@ -95,32 +101,242 @@ def GetParams(self):
             padding="SAME",
             name="conv")
         c1 = constant_op.constant(
-            np.random.randn(input_dims[0], 12, 12, 6), dtype=dtype)
-        p = conv * c1
+            np.random.randn(input_dims[0], 12, 12, 6), dtype=dtype, name="c1")
+        p = math_ops.mul(conv, c1, name="mul")
```

Comment:
What's the reason of using `math_ops` here instead of operators like `*`?

Reply:
Because we need a name for `expected_engines` in the following.

Reply:
Okay. Nice.